tests/e2e/pact/broker/verify.ts

```diff
@@ -94,15 +102,17 @@ async function verifyPrepare(): Promise<any> {
 async function verifyProcess(): Promise<any> {
     const pactGroups =
       process.env.PACT_PROVIDER_URL.includes("sandbox")
-      ? PactGroups.filter(g => g !== "failures")
+      ? PactGroups
+          .filter(g => g !== "failures")
+          .filter(g => !g.includes("-cancel"))
       : PactGroups
 
     await pactGroups.reduce(async (promise, group) => {
       await promise
       endpoint = "process"
       pactGroup = group
       resetBackOffRetryTimer()
-      await verifyWith2Retries()
```

Comment:
If process message smoke-test fails the first time it will almost definitely fail the second time as some prescriptions will have been sent and will now be considered "duplicate", or fail a subsequent time as a result of a cached response for the x-request-id in spine. This change gives us a cleaner log to look at when something goes wrong